draft-pauly-intarea-proxy-config-pvd.md

```diff
@@ -37,7 +37,7 @@ HTTP proxies that use the CONNECT method {{Section 9.3.6 of !HTTP=RFC9110}}
 (often referred to as "forward" proxies) allow clients to open connections to
 hosts via a proxy. These typically allow for TCP stream proxying, but can also support
 UDP proxying {{!CONNECTUDP=RFC9298}} and IP packet proxying
-{{!CONNECTIP=I-D.ietf-masque-connect-ip}}. Such proxies are not just defined as
+{{!CONNECTIP=RFC9484}}. Such proxies are not just defined as
```

Comment:
nit: can we do `CONNECT-IP` instead of `CONNECTIP` to match how the other MASQUE documents reference each other? (same for UDP and TCP)

Reply:
Fixed